pgmpy/models/BayesianModel.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, ebunch=None):
             self.add_edges_from(ebunch)
```

Comment:
@palashahuja Don't change things in BayesianModel. Better simply inherit from DirectedGraph.